### DISCUSSION TITLE
contribute?

### DISCUSSION THREAD

**Discussion by s3nix-sys** [samuelmaddock/electron-browser-shell#98] [General]
i see a good path for this project, i would like to contribute and add features and help at no extra cost to you, i could add the manifest v3 support and stuff like that if youd like?, or i coould just push features ither one

**Comment by samuelmaddock** [comment]
Manifest V3 support is in progress  https://github.com/samuelmaddock/electron-browser-shell/issues/38

It depends on service workers preload scripts which will take time to land in Electron.

Pull requests are open though so you're welcome to create one for anything other than Manifest v3.
